src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -1482,6 +1482,21 @@ def __init__(self, config):
         checkpoint="microsoft/DialogRPT-updown",
         output_type=TokenClassifierOutput,
         config_class=_CONFIG_FOR_DOC,
+        expected_loss=0.0,
+        expected_output=[
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+        ],
```

Comment:
This is not a helpful example at all, also, this model seems to be a sequence classification model according to its model card. The loss at 0.0 is very weird especially.

Reply:
Indeed, it is a text (sequence) classification model.

@ArEnSc Could you try if the following GPT2 **token** classification model ? 

https://huggingface.co/brad1141/gpt2-finetuned-comp2